utils/context_resolutor.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * A function that receives a string with a dot key foo.v1.v2
+ * and searches in the github context of the Action for its value. 
+ *
+ * If nothing is found it throws an Exception. 
+ *
+ * Args:
+ *   - key(string): a dotted key to search for its value
+ * Returns:
+ *   - value(string): the correspondant value
+ */
+
+const github = require('@actions/github');
+
+module.exports = function(key){
+
+  let resolved = false
+
+  for(const level of key.split(/\./)){
+
+    if(!resolutor){
+
+      resolved = __getResolutor(level)
+    
+    }
+    else{
+    
+      if(!resolved[l]){
+
+        throw `COULD NOT RESOLVE ${key}`
+      }
+
+      resolved = resolved[l]
+
+    }
+
+
+  }
+
+  return resolved
+
+}
+
+  function __getResolutor(level){
+
+    switch(level){
+
+      case "secrets":
+        return github.context.secrets
+      case "env"
+        return github.context.env
+      default:
```

Comment:
Are you sure that the secrets and env contexts are available in this way?
https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts

Reply:
Not sure at all, I was going to test it in this way an make further corrections.

Reply:
Fixed for the env object.

Reply:
`${{secrets}}` cannot be interpolated directly in an action execution, they must be passed from the workflow, as env or input variable. Update related issue #12
https://docs.github.com/en/actions/security-guides/encrypted-secrets#accessing-your-secrets
![image](https://user-images.githubusercontent.com/1025564/153001433-5283fab8-3c4e-4727-8d42-85a68b008125.png)